src/main/router.ts

```diff
@@ -179,8 +179,11 @@ export class Router {
     this.router.add({ method: "post", path: `${apiPrefix}/metrics` }, MetricsRoute.routeMetrics);
     this.router.add({ method: "get", path: `${apiPrefix}/metrics/providers` }, MetricsRoute.routeMetricsProviders);
 
-    // Port-forward API
-    this.router.add({ method: "post", path: `${apiPrefix}/pods/{namespace}/{resourceType}/{resourceName}/port-forward/{port}` }, PortForwardRoute.routePortForward);
+    // Port-forward API (the container port and local forwarding port are obtained from the query parameters)
+    this.router.add({ method: "post", path: `${apiPrefix}/pods/port-forward/{namespace}/{resourceType}/{resourceName}` }, PortForwardRoute.routePortForward);
+    this.router.add({ method: "get", path: `${apiPrefix}/pods/port-forward/{namespace}/{resourceType}/{resourceName}` }, PortForwardRoute.routeCurrentPortForward);
+    this.router.add({ method: "get", path: `${apiPrefix}/pods/port-forwards` }, PortForwardRoute.routeAllPortForwards);
```

Comment:
I think so. Looks more consistent to me. Port-forwards are associated with pods (containers in the pods). Or we could lose `/pods` altogether...?

Reply:
No I don't think we should loose the `/pods`... It just feels odd to me that this one route is about all clusters and the others are about single clusters. But I don't know how to remedy that. So lets leave it.

Reply:
Hmm, they're not explicitly about single clusters but maybe they should be (we could add cluster-id as a path segment or query param?). Anyway, I think this can be resolved when we add a global page (port-forwards in the catalog?). This api is strictly for internal ipc, right? We can change it anytime.

Reply:
Yes we can change it at any time, technically any extension could just use it but we don't really expose it.

I meant cluster scoped because you use the attached cluster object in the other route handlers.